generator/internal/descriptor_utils.cc

```diff
@@ -236,30 +236,32 @@ ParameterCommentSubstitution substitutions[] = {
     //
     // Doxygen thinks these are unrecognized HTML tags. To work around this, we
     // remove the line breaks, so the code quotes occupy a single line.
-    {"agents/\n <Agent ID>", "agents/<Agent ID>"},
-    {"<Agent\n ID>", "<Agent ID>"},
-    {"<Agent ID>/\n testCases/", "<Agent ID>/testCases/"},
-    {"<Agent ID>/\n environments/", "<Agent ID>/environments/"},
+    // {"agents/\n <Agent ID>", "agents/<Agent ID>"},
```

Comment:
I led you astray. We should delete these lines, not comment them out.

Reply:
Oh, I see. How did you find out which substitution to remove to make the library generation work so quickly?

Reply:
It looks like `publish-docs` is still failing.

They changed the comments upstream, but I bet they missed a spot somewhere.

https://github.com/googleapis/googleapis/commit/dc219758b92d84a58e46228cc32654027e31d8b3

There could be more, but it is probably forgetting to escape these format strings. Then `doxygen` sees what it thinks is an HTML tag and freaks out.

https://github.com/googleapis/googleapis/blob/dc219758b92d84a58e46228cc32654027e31d8b3/google/cloud/dialogflow/cx/v3/experiment.proto#L264

We are going to need to add new substitutions to appease `doxygen`. It also good practice to find the offending comments in google3 and send a CL to update them.

Reply:
> These iterations are not fun.

Tip: Generating the documentation for all the libraries is sloooooooooow. I would locally set `GOOGLE_CLOUD_CPP_ENABLE=dialogflow_cx` for faster iterations.

https://github.com/googleapis/google-cloud-cpp/blob/5c3ffb5b310cda9cd40107d9c366fe914d925c63/ci/cloudbuild/builds/publish-docs.sh#L45

Reply:
Thank you! That is very clear.